RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -145,6 +145,52 @@ boolean auto_fightLocketMonster(monster mon, boolean speculative)
 
 }
 
+boolean auto_haveGreyGoose()
+{
+	if(auto_have_familiar($familiar[Grey Goose]))
+	{
+		return true;
+	}
+	return false;
+}
+
+int gooseExpectedDrones()
+{
+	if(!auto_haveGreyGoose()) return 0;
+	if(my_familiar() == $familiar[Grey Goose])
+	{
+		set_property("auto_gooseExpectedDrones", familiar_weight($familiar[Grey Goose]) - 5);
```

Comment:
what does this do in the event you use fidoxene, which "changes" the fam's base weight? does familiar_weight still grab true base weight? i do not remember off top of mind but we want to be careful not to dramatically overestimate

Reply:
I think we can care about this when we actually use Fidoxene.

Which is to say, never.